hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettag/targettype/AddTargetTypeWindowController.java

```diff
@@ -81,6 +86,9 @@ protected Class<? extends ProxyIdentifiableEntity> getParentEntityClass() {
 
     @Override
     protected boolean isEntityValid(final ProxyTargetType entity) {
-        return validator.isEntityValid(entity, () -> targetTypeManagement.getByName(entity.getName()).isPresent());
+        return validator.isEntityValid(entity,
+                () -> contextAware.runAsTenant(
+                        contextAware.getCurrentTenant(),
+                        () -> targetTypeManagement.getByName(entity.getName()).isPresent()));
```

Comment:
contextAware#runAsTenant based on contextAware#getCurrentTenant. I assume this will temporary deactivate the access restrictions. However this looks confusing for me, since the affect is not recognizable

Reply:
Maybe we should wrap protected `boolean isEntityValid(final ProxyTargetType entity)` per default in an unrestricted execution